admin_guide/managing_networking.adoc

```diff
@@ -144,6 +144,36 @@ $ systemctl restart atomic-openshift-master-api atomic-openshift-master-controll
 
 ----
 
+[[admin-guide-managing-networking-ingress]]
+== Setting up the HAProxy router to accept ingress objects
+
+By default, the HAProxy router only accepts route objects. You can set up the
```

Comment:
In order to describe the comparison, I mentioned the router only accepts route objects. Is this accurate?

Reply:
That is correct.